packages/fluentui/react-builder/src/config.tsx

```diff
@@ -511,6 +558,10 @@ export const resolveDrop = (newChild: JSONTreeElement, parent: JSONTreeElement,
   console.log('config:resolveDrop RESULT', JSON.parse(JSON.stringify({ source: newChild, target: parent })));
 };
 
+export const isValidDrop = (element: JSONTreeElement) => {
+  return element.type === 'div' || Object.keys(element.props).length === 0 || element.props?.children;
```

Comment:
if `element.props` is null or undefined won't `Object.keys(element.props)` throw ?

Reply:
It should never be undefined, in `resolveDraggingElement` we are retruning 
`return {
    uuid: getUUID(),
    $$typeof: 'Symbol(react.element)',
    type: displayName,
    moduleName: module,
    displayName,
    props: { children: [] },
    ...jsonTreeElement,
  };`
Even when you add component via just code writing the props is set to {}. Though it might be a good idea to check whether there are props just in case the code changes.